### ISSUE TITLE
No colours but escape chars in case of HTTP tracing

### ISSUE THREAD

**Issue by mdeknowis** [IBM-Cloud/ibm-cloud-cli-sdk#135]
Hello,

we are creating our own bx plugin. We use the Http logging according
https://github.com/IBM-Cloud/ibm-cloud-cli-sdk/blob/master/docs/plugin_developer_guide.md#41-http-tracing

But we realized, that instead of colors, we see escaped control bytes.
The normal logging coloring is working (Warn, Fatal...)

---- SNIP ---
Retrieving...

[1;39mREQUEST:[0m [2018-06-21T13:51:50+02:00]
POST /auth/realms/integration/protocol/openid-connect/token HTTP/1.1
Host: myhost.net
Content-Type: application/x-www-form-urlencoded

...

[1;39mRESPONSE:[0m [2018-06-21T13:51:51+02:00] [1;39mElapsed:[0m 416ms
HTTP/1.1 200 OK
Content-Length: 2501
...

[1;39mREQUEST:[0m [2018-06-21T13:51:51+02:00]
POST /api/r HTTP/1.1
...

[1;39mRESPONSE:[0m [2018-06-21T13:51:51+02:00] [1;39mElapsed:[0m 64ms
HTTP/1.1 401
...
--- SNAP ---

Regards,
Michael

**Comment by weieigao** [comment]
What's your platform? Was the plugin running on a real terminal?

**Comment by mdeknowis** [comment]
Hello,

I was running it in Windows Shell (cmd). Funny, it's working now:
![grafik](https://user-images.githubusercontent.com/23213544/41958579-c27383b6-79ea-11e8-9ddb-439dc6f79205.png)

But instead of the control characters, there are just no colors. I don't know, if this is by purpose.

In general colors are working:
![grafik](https://user-images.githubusercontent.com/23213544/41958626-e9576844-79ea-11e8-85a8-fde8ad820017.png)

Please close this, if no colors should appear.

**Comment by weieigao** [comment]
@mdeknowis  no, it's not by purpose... We will check the code, thanks for reporting the issue.
